src/components/heading/heading.component.tsx

```diff
@@ -20,6 +20,7 @@ import {
 } from "./heading.style";
 import useLocale from "../../hooks/__internal__/useLocale";
 
+export type HeadingTypes = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

Comment:
```suggestion
export type HeadingType = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
```

minor thing but I think the name of the type should be singular - a value of this type represents a single `HeadingType`, it shouldn't be plural just because there are a range of options for values it can take. Note that this is consistent with the `IconType` type used in various components (although I haven't checked if we have the plural naming elsewhere - but I think it should be singular and `IconType` is a widely used type in Carbon).